[2025-02-16T13:22:38.939+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-16T13:22:39.467+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fourth_dag.task2 scheduled__2025-02-14T21:33:00+00:00 [queued]>
[2025-02-16T13:22:39.715+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fourth_dag.task2 scheduled__2025-02-14T21:33:00+00:00 [queued]>
[2025-02-16T13:22:39.716+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-16T13:22:40.804+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): task2> on 2025-02-14 21:33:00+00:00
[2025-02-16T13:22:40.843+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=5370) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-16T13:22:40.847+0000] {standard_task_runner.py:72} INFO - Started process 5390 to run task
[2025-02-16T13:22:40.892+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fourth_dag', 'task2', 'scheduled__2025-02-14T21:33:00+00:00', '--job-id', '878', '--raw', '--subdir', 'DAGS_FOLDER/04_fourth_dag.py', '--cfg-path', '/tmp/tmp4q5fwhlz']
[2025-02-16T13:22:40.894+0000] {standard_task_runner.py:105} INFO - Job 878: Subtask task2
[2025-02-16T13:22:41.660+0000] {task_command.py:467} INFO - Running <TaskInstance: fourth_dag.task2 scheduled__2025-02-14T21:33:00+00:00 [running]> on host ab74d4d90d32
[2025-02-16T13:22:42.605+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fourth_dag' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T21:33:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-14T21:33:00+00:00'
[2025-02-16T13:22:42.627+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-16T13:22:43.093+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-16T13:22:43.210+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 5']
[2025-02-16T13:22:43.271+0000] {subprocess.py:99} INFO - Output:
[2025-02-16T13:22:51.222+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-16T13:22:51.412+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-16T13:22:51.444+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fourth_dag, task_id=task2, run_id=scheduled__2025-02-14T21:33:00+00:00, execution_date=20250214T213300, start_date=20250216T132239, end_date=20250216T132251
[2025-02-16T13:22:56.241+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-16T13:22:56.344+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-16T13:22:56.356+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
